### PR TITLE
[DOCS] Refresh kv docs

### PR DESCRIPTION
**Pull request by schavis**
Breaks the current [key/value v2](https://developer.hashicorp.com/vault/docs/secrets/kv/kv-v2) plugin docs into individual guides and cookbook entries with instructions for CLI, GUI, and API

Preview: [Key/Value v2 plugin](https://vault-mh95ulejr-hashicorp.vercel.app/vault/docs/secrets/kv/kv-v2)

### CODE REVIEW

File: website/content/docs/secrets/kv/kv-v2/setup.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+- Enter a unique path for the plugin and provide the relevant configuration
+  data.
+
+</Tab>
+
+</Tabs>
+
+
+
+## Step 2: Create an ACL policy file
```
Comment (yhyakuna) [inline website/content/docs/secrets/kv/kv-v2/setup.mdx:68]: Maybe you can mention the [`-output-policy` flag usage](https://developer.hashicorp.com/vault/docs/commands#construct-the-required-vault-policy) somewhere? I helps users to write/verify policies.

**Review by yhyakuna — APPROVED** [review]
Added a minor suggestion. Regardless, it looks great.

File: website/content/docs/secrets/kv/kv-v2/setup.mdx (hunk 1 above)
Comment (schavis) [reply website/content/docs/secrets/kv/kv-v2/setup.mdx:68]: That's a great idea. I'll add that today

File: website/content/docs/secrets/kv/kv-v2/cookbook/custom-metadata.mdx [hunk 2]
```diff
@@ -0,0 +1,148 @@
+---
+layout: docs
+page_title: Write custom metadata
+description: >-
+   Write custom metadata fields to your kv v2 plugin.
+---
+
+# Write custom metadata in key/value v2 
+
+Write custom metadata to a `kv` v2 secret path for ???.
```
[the remark this answers is not in the document]
Comment (schavis) [reply website/content/docs/secrets/kv/kv-v2/cookbook/custom-metadata.mdx:10]: D'oh! Yes that's a placeholder. I'll get that fixed 😅

**Pull request merged**